central/alert/datastore/datastore_test.go

```diff
@@ -80,17 +80,17 @@ func (s *alertDataStoreTestSuite) TestSearchRawAlerts() {
 }
 
 func (s *alertDataStoreTestSuite) TestSearchListAlerts() {
-	s.searcher.EXPECT().SearchListAlerts(s.hasReadCtx, &v1.Query{}).Return(alerttest.NewFakeListAlertSlice(), errFake)
+	s.searcher.EXPECT().SearchListAlerts(s.hasReadCtx, &v1.Query{}, true).Return(alerttest.NewFakeListAlertSlice(), errFake)
```

Comment:
You should probably add some tests using the new wrapper.

Reply:
+1